app/src/main/java/com/mertceyhan/bitcoinmarket/features/market/di/MarketModule.kt

```diff
@@ -11,11 +11,11 @@ import dagger.Binds
 import dagger.Module
 import dagger.Provides
 import dagger.hilt.InstallIn
-import dagger.hilt.android.components.ViewModelComponent
+import dagger.hilt.components.SingletonComponent
 import retrofit2.Retrofit
 
 @Module
-@InstallIn(ViewModelComponent::class)
+@InstallIn(SingletonComponent::class)
```

Comment:
Is there a reason for this change?

Reply:
I injected MarketInformationUseCase to MarketWidgetReceiver. Because ViewModelComponent scope restricted to me, I changed it as SingletonComponent.